espnet2/asr/espnet_model.py

```diff
@@ -60,16 +60,28 @@ def __init__(
         sym_space: str = "<space>",
         sym_blank: str = "<blank>",
         extract_feats_in_collect_stats: bool = True,
+        sos_index: int = -1,
```

Comment:
Can you also use `sym_sos` defined in the latest [espnet](https://github.com/espnet/espnet/blob/ca2193d54031eb469b5a02cf60c6ff0b185d7ca4/espnet2/asr/espnet_model.py#L64) to index the sos and eos symbols?